hack/jsonformat.go

```diff
@@ -13,8 +13,8 @@ func main() {
 		log.Fatal("Usage: jsonformat.go <filename>")
 	}
 
-	byt, err := ioutil.ReadFile(os.Args[1])
-	if err != nil {
+	byt, readErr := ioutil.ReadFile(os.Args[1])
```

Comment:
Why is this shadowing.  Redefining err is not an anti pattern in the same level.  This is anti our conventions.